examples/xds-management-server/cmd/controller.go

```diff
@@ -0,0 +1,94 @@
+package main
+
```

Comment:
Thought for next time: The Agones integration could have been a seperate PR after the file provider had been merged. Might have cut down on the 4,000 lines 😃